### ISSUE TITLE
Websockets aren't working with postgres

### ISSUE THREAD

**Issue by grovecj** [CSSE497/pathfinder-server#18]
I get the following error when trying to create an empty cluster from the angular app. The request works using an in-memory database

```
[info] - application - Received web socket crud request Create(Cluster,{})
[error] - akka.actor.OneForOneStrategy - ERROR executing DML bindLog[] error[ERROR: syntax error at or near ")"\n   Position: 22]
javax.persistence.PersistenceException: ERROR executing DML bindLog[] error[ERROR: syntax error at or near ")"\n   Position: 22]
    at com.avaje.ebeaninternal.server.persist.dml.DmlBeanPersister.execute(DmlBeanPersister.java:90) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.persist.dml.DmlBeanPersister.insert(DmlBeanPersister.java:57) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.persist.DefaultPersistExecute.executeInsertBean(DefaultPersistExecute.java:59) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.core.PersistRequestBean.executeNow(PersistRequestBean.java:446) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.core.PersistRequestBean.executeOrQueue(PersistRequestBean.java:480) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:261) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.persist.DefaultPersister.insert(DefaultPersister.java:211) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.core.DefaultServer.insert(DefaultServer.java:1537) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.core.DefaultServer.insert(DefaultServer.java:1530) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebean.Model.insert(Model.java:226) ~[avaje-ebeanorm-4.6.2.jar:na]
    at io.pathfinder.data.EbeanCrudDao$$anonfun$create$1.apply(EbeanCrudDao.scala:17) ~[classes/:na]
    at io.pathfinder.data.EbeanCrudDao$$anonfun$create$1.apply(EbeanCrudDao.scala:16) ~[classes/:na]
    at scala.Option.map(Option.scala:146) ~[scala-library.jar:na]
    at io.pathfinder.data.EbeanCrudDao.create(EbeanCrudDao.scala:15) ~[classes/:na]
    at io.pathfinder.websockets.controllers.WebSocketCrudController.receive(WebSocketCrudController.scala:27) ~[classes/:na]
    at io.pathfinder.websockets.WebSocketActor$$anonfun$receive$1$$anonfun$applyOrElse$1.apply(WebSocketActor.scala:29) ~[classes/:na]
    at io.pathfinder.websockets.WebSocketActor$$anonfun$receive$1$$anonfun$applyOrElse$1.apply(WebSocketActor.scala:29) ~[classes/:na]
    at scala.Option.flatMap(Option.scala:171) ~[scala-library.jar:na]
    at io.pathfinder.websockets.WebSocketActor$$anonfun$receive$1.applyOrElse(WebSocketActor.scala:29) ~[classes/:na]
    at akka.actor.Actor$class.aroundReceive(Actor.scala:467) ~[akka-actor_2.11-2.3.13.jar:na]
    at io.pathfinder.websockets.WebSocketActor.aroundReceive(WebSocketActor.scala:22) ~[classes/:na]
    at akka.actor.ActorCell.receiveMessage(ActorCell.scala:516) [akka-actor_2.11-2.3.13.jar:na]
    at akka.actor.ActorCell.invoke(ActorCell.scala:487) [akka-actor_2.11-2.3.13.jar:na]
    at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:238) [akka-actor_2.11-2.3.13.jar:na]
    at akka.dispatch.Mailbox.run(Mailbox.scala:220) [akka-actor_2.11-2.3.13.jar:na]
    at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:397) [akka-actor_2.11-2.3.13.jar:na]
    at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
    at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
    at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
    at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: org.postgresql.util.PSQLException: ERROR: syntax error at or near ")"
  Position: 22
    at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2103) ~[postgresql-9.1-901-1.jdbc4.jar:na]
    at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1836) ~[postgresql-9.1-901-1.jdbc4.jar:na]
    at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:257) ~[postgresql-9.1-901-1.jdbc4.jar:na]
    at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:512) ~[postgresql-9.1-901-1.jdbc4.jar:na]
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:388) ~[postgresql-9.1-901-1.jdbc4.jar:na]
    at org.postgresql.jdbc2.AbstractJdbc2Statement.executeUpdate(AbstractJdbc2Statement.java:334) ~[postgresql-9.1-901-1.jdbc4.jar:na]
    at com.zaxxer.hikari.proxy.PreparedStatementProxy.executeUpdate(PreparedStatementProxy.java:61) ~[HikariCP.jar:na]
    at com.zaxxer.hikari.proxy.PreparedStatementJavassistProxy.executeUpdate(PreparedStatementJavassistProxy.java) ~[HikariCP.jar:na]
    at com.avaje.ebeaninternal.server.type.DataBind.executeUpdate(DataBind.java:56) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.persist.dml.InsertHandler.execute(InsertHandler.java:128) ~[avaje-ebeanorm-4.6.2.jar:na]
    at com.avaje.ebeaninternal.server.persist.dml.DmlBeanPersister.execute(DmlBeanPersister.java:80) ~[avaje-ebeanorm-4.6.2.jar:na]
    ... 29 common frames omitted
```

**Comment by aj-michael** [comment]
@danhanson any ideas? I've been investigating this a bit today and have come up with nothing.

**Comment by robinsdm** [comment]
This is probably an error related to inserting stuff that isn't in the proper format. It looks like there is some error related to a ")" not being in the right place or something near one.

**Comment by aj-michael** [comment]
Hmmm. I am able to read and write vehicles successfully though websockets, but I get the same error for clusters.

**Comment by aj-michael** [comment]
To clarify, this is not a problem with websockets, it is a problem with the Cluster mode.

**Comment by aj-michael** [comment]
Fixed this in #35. The problem is a bug in Ebeans that generates invalid Postgres syntax if your table has no fields. So I added the authenticationToken field for clusters.

**Comment by aj-michael** [comment]
Actual time: 1.5 hours.
